libs/shared/typescript/charts-ngx/src/lib/boxplot/boxplot.directive.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  Directive,
+  ElementRef,
+  Input,
+  OnChanges,
+  OnDestroy,
+  OnInit,
+} from '@angular/core';
+import {
+  BoxplotChart,
+  BoxplotProps,
+  CategoryBoxplotSummary,
+  CategoryPoint,
+} from '@sagebionetworks/shared/charts';
+
+@Directive({
+  selector: '[sageBoxplot]',
```

Comment:
might be a nitpick, but should this be named sageBoxPlot?

Reply:
The [Wikipedia article](https://en.wikipedia.org/wiki/Box_plot) references both "box plot" and "boxplot", but "box plot" seems to be more common, so I'll change the name to `sageBoxPlot`!

Reply:
Good point @tschaffter! I'll stick with "boxplot".